server/controllers.js

```diff
@@ -0,0 +1,70 @@
+const request = require('request');
+
+const createJsonController = endpoint => async (req, res) => {
+
```

Comment:
Creating controllers with a function is pretty clever. Im probably gonna steal that idea